Source/Core/DiscIO/WbfsBlob.cpp

```diff
@@ -47,6 +47,15 @@ WbfsFileReader::~WbfsFileReader()
 {
 }
 
+std::unique_ptr<BlobReader> WbfsFileReader::CopyReader() const
+{
+  auto retval = std::unique_ptr<WbfsFileReader>(
+      new WbfsFileReader(std::move(*m_files[0].file.Duplicate("r")), ""));
```

Comment:
The code would be cleaner if you added an overload of the constructor that didn't take a path parameter and didn't try to scan for additional files. Passing in a path that "should" be invalid and counting on OpenAdditionalFiles failing seems kind of fragile, even though I suppose it does work in practice.

Reply:
Proposing a possible alternative here, making the path field optional and skipping OpenAdditionalFiles if it's empty. Your thoughts?

Reply:
I suppose that works.